cli/program_state.rs

```diff
@@ -65,11 +68,50 @@ impl ProgramState {
     let dir = deno_dir::DenoDir::new(custom_root)?;
     let deps_cache_location = dir.root.join("deps");
     let http_cache = http_cache::HttpCache::new(&deps_cache_location);
+
+    let default_ca_stores = vec!["mozilla".to_string()];
+    let supported_ca_stores = vec!["mozilla".to_string(), "system".to_string()];
+
+    let mut root_store = RootCertStore::empty();
+    let ca_stores: Vec<String> = flags
+      .ca_stores
+      .clone()
+      .or_else(|| {
+        let env_ca_store = env::var("DENO_TLS_CA_STORE");
+        match env_ca_store {
+          Ok(arg) => Some(arg.split(',').map(|s| s.to_string()).collect()),
+          _ => None,
+        }
```

Comment:
Minor thing but you can probably shorten this to:
```suggestion
        let env_ca_store = env::var("DENO_TLS_CA_STORE").ok()?;
        Some(env_ca_store.split(',').map(|s| s.to_string()).collect())
```
Aside: I'd trim whitespace because people are bound to write `"system, mozilla"` and then get confused when it doesn't work.

Reply:
@bnoordhuis if I did `.ok()?` and the environment variable is not set that would return in an error, no? I want it to be ok to not have the variable, which would then fallback to the default behavior.

Reply:
No you're right that works perfect and is cleaner. Thanks.